### ISSUE TITLE
Logits and RepetitionPenalty implementation

### ISSUE THREAD

**Issue by ahmedjawedaj** [kingoflolz/mesh-transformer-jax#57]
Hello @trisongz Hope you're doing well, do you have any plan to Implement `Logits and RepetitionPenalty `
something similar implemented here
https://huggingface.co/transformers/_modules/transformers/generation_logits_process.html

Thankyou.

**Comment by kingoflolz** [comment]
There are no plans to add fancy sampling options to this repository. The huggingface repository provides a unified interface for that and has a pending PR for GPT-J (https://github.com/huggingface/transformers/pull/12243)
